cfme/services/requests.py

```diff
@@ -197,6 +197,30 @@ def debug_requests():
             logger.debug(' {}'.format(row))
 
 
+def go_to_request_page(cells):
+    """Finds the request and opens the page
```

Comment:
This function seems to perhaps be wrongly named? I can't see a click, just a find and return True. Yet the function below does click?